### DISCUSSION TITLE
Docs for Container, Row, Col

### DISCUSSION THREAD

**Discussion by kmarler18** [reactstrap/reactstrap#2763] [Q&A] [answered]
Hey Gang,

I found [this ](https://reactstrap.github.io/)super helpful resource explaining some of the ReactStrap components; however, I'm not able to locate the documentation for the Container, Col, or Row components. Maybe I'm missing something. Thanks in advance!

**Comment by merpeople** [comment, marked as answer]
Check [Layout](https://reactstrap.github.io/?path=/docs/components-layout--layout-row-cols) in the Docs.
